projects/admin/src/app/document-editor/document-editor.component.ts

```diff
@@ -0,0 +1,59 @@
+import { Component, OnInit, Inject } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
+import { EditorComponent } from '@rero/ng-core';
+import { FrameworkLibraryService } from 'angular6-json-schema-form';
+import { RecordService } from '@rero/ng-core';
+import { TranslateService } from '@ngx-translate/core';
+import { ToastrService } from 'ngx-toastr';
+import { WidgetLibraryService } from 'angular6-json-schema-form';
+import { Location } from '@angular/common';
+import { CustomBootstrap4Framework } from '@rero/ng-core';
+import { ExtraService } from '../service/extra.service';
+import { marker as _ } from '@biesbjerg/ngx-translate-extract-marker';
+
+
+@Component({
+  selector: 'admin-document-editor',
+  templateUrl: './document-editor.component.html',
+  styleUrls: ['./document-editor.component.css']
+})
+export class DocumentEditorComponent extends EditorComponent implements OnInit {
+
+  constructor(
+    @Inject(CustomBootstrap4Framework) bootstrap4framework,
+    route: ActivatedRoute,
+    recordService: RecordService,
+    widgetLibrary: WidgetLibraryService,
+    translateService: TranslateService,
+    location: Location,
+    toastrService: ToastrService,
+    frameworkLibrary: FrameworkLibraryService,
+    private extraService: ExtraService,
+    private toastService: ToastrService
+  ) {
+    super(
+      bootstrap4framework, route, recordService, widgetLibrary, translateService, location, toastrService, frameworkLibrary);
+  }
+
+  ngOnInit() {
+    super.ngOnInit();
+  }
+
+  importFromEan(ean) {
```

Comment:
can you start to create documentation as ng-core?

Reply:
Do you mean to documente the function?

Have you any example in ng-core?

Reply:
Yes, please do not hesitate to have a look to the github repo of `ng-core`.